pages/common/doctl-databases-db.md

```diff
@@ -0,0 +1,24 @@
+# doctl databases db
+
+> Manage specific databases that are served by a database cluster.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/db/>.
+
+- Run a doctl databases db command with an access token:
+
+`doctl databases db [command] --access-token {{ access_token }}`
```

Comment:
```suggestion
`doctl databases db [command] --access-token {{access_token}}`
```

No leading and trailing spaces on placeholder names, [like our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax).